applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -518,6 +518,9 @@ def AddSpheresVariables(self, model_part, DEM_parameters):
         if self.DEM_parameters["PostControlModule"].GetBool():
             model_part.AddNodalSolutionStepVariable(TARGET_STRESS)
             model_part.AddNodalSolutionStepVariable(REACTION_STRESS)
+            model_part.AddNodalSolutionStepVariable(SMOOTHED_REACTION_STRESS)
+            model_part.AddNodalSolutionStepVariable(ELASTIC_REACTION_STRESS)
+            model_part.AddNodalSolutionStepVariable(SMOOTHED_ELASTIC_REACTION_STRESS)
```

Comment:
do you need all of them to be historical?

Reply:
This is for post-processing... Maybe we could print that information at the conditions instead of at the nodes, but I don't know if this change would be easy in the DEMApplication. Furthermore, some information would have to be printed at DEM elements and some at DEM conditions so it could be a bit messy..

Reply:
you can also use non-historical Variables on the Nodes (same sysntax as for elements & conditions)

Reply:
But are these kind of variables printed in GiD? I actually didn't know it.

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/e3b70ebacf6d5fd22990a304de7939054c0e07b3/kratos/python_scripts/gid_output_process.py#L40-L41

`nodal_results`: historical variables
`nodal_nonhistorical_results`: non-historical variables

Reply:
vtk supports it too

Reply:
Ok, that's good. Then I'll try to use them.

Reply:
Those new nodal variables are now non-historical. (I just left the minimum number of nodal historical variables for the CM)